sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1788,6 +1788,15 @@ static void _send(gnrc_netif_t *netif, gnrc_pktsnip_t *pkt, bool push_back)
 #endif /* IS_USED(MODULE_GNRC_NETIF_PKTQ) */
 }
 
+void gnrc_netif_bhp_irq_handler(void *ctx)
+{
+    gnrc_netif_bhp_ctx_t *c = ctx;
+    if (&c->netif.evq) {
```

Comment:
Isn't  this always true?

Reply:
uf, of course. I forgot to remove that `if`